modules/cnf-topology-aware-lifecycle-operator-apply-policies.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-apply-policies_{context}"]
+= Applying update policies to managed clusters
+
+The Topology Aware Lifecycle Operator remediates a set of managed policies in the specified clusters. Then, the Operator makes enforced copies of the managed {rh-rhacm} policies. Each copied policy has its own corresponding {rh-rhacm} placement rule and {rh-rhacm} placement binding. One by one, the Operator adds each cluster in the current batch to the placement rule that corresponds with the applicable managed policy. After a batch is updated, all clusters are removed from the placement rules associated with the copied policies. Then, the update of the next batch starts.
```

Comment:
```suggestion
The Topology Aware Lifecycle Operator remediates a set of managed policies in the specified clusters. TALO makes enforced copies of the managed {rh-rhacm} policies. Each copied policy has its own corresponding {rh-rhacm} placement rule and {rh-rhacm} placement binding. One by one, TALO adds each cluster from the current batch to the placement rule that corresponds with the applicable managed policy. After a batch is updated, all clusters are removed from the placement rules associated with the copied policies. Then, the update of the next batch starts.
```

Reply:
Another note to make here is that if a cluster is already compliant with a managed policy, TALO won't try to reapply that policy on the cluster, but it will skip it and move to the next policy for which the current cluster is NonCompliant.

Reply:
Also, we should clearly state that the managed policies should be in inform.

Reply:
Added `inform` and info about TALO skipping clusters. Plus divided into to paragraphs for readability.
> The Topology Aware Lifecycle Operator remediates a set of **`inform`** policies for the clusters specified in the `ClusterGroupUpgrade` CR. Then, TALO makes enforced copies of the managed {rh-rhacm} policies. Each copied policy has its own corresponding {rh-rhacm} placement rule and {rh-rhacm} placement binding. 
>One by one, TALO adds each cluster from the current batch to the placement rule that corresponds with the applicable managed policy. If a cluster is compliant with a policy, TALO skips applying the policy on the compliant cluster. TALO then moves on to applying the next policy, with which the cluster is non-compliant. After a batch is updated, all clusters are removed from the placement rules associated with the copied policies. Then, the update of the next batch starts.